src/App.css

```diff
@@ -21,7 +21,7 @@
   align-items: center;
```

Comment:
### Optional
- [ ] It would be better if we remove all the React boilerplate code which is not needed for the project :woman_technologist: